<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>99</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Enter Text</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Text:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>180</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="done_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
